ui/vuetifyx/vuetifyxjs/src/lib/Datetimepicker.vue

```diff
@@ -149,10 +151,13 @@ const init = () => {
   let initDateTime = parse(props.modelValue, dateTimeFormat.value, new Date())
   date.value = initDateTime
   time.value = format(initDateTime, DEFAULT_TIME_FORMAT)
+  dateOfPicker.value = date.value
+  timeOfPicker.value = time.value
 }
 
 const okHandler = (isActive: Ref) => {
   resetPicker(isActive)
+  date.value = dateOfPicker.value
```

Comment:
You define two aliases for the original variable, but at the end you assign it back to the original value. What's the point of that?

Reply:
It is just to prevent the value of the text-field from being updated before clicking OK or Clear. Otherwise, if the dialog disappears early by clicking outside the dialog, the value of the text-field has been updated, which is not as expected.

xxxOfPicker: Intermediate state
xxx: Confirmed state

Reply:
okay, if the original def() exp. date or time  is not reactive by template anymore, let them be a `let` value is better way, it depends on you.